packages/0x.js/src/web3_wrapper.ts

```diff
@@ -72,8 +74,9 @@ export class Web3Wrapper {
     public getNetworkId(): number {
         return this.networkId;
     }
-    public async getContractInstanceFromArtifactAsync<A extends Web3.ContractInstance>(artifact: Artifact,
-                                                                                       address?: string): Promise<A> {
+    public async getContractInstanceFromArtifactAsync(
```

Comment:
Hm... Should this belong in `wrappedWeb3`? It calls one web3 method but doesn't **really** have much to do with web3. Thoughts?

Reply:
Yeah... Do you think it'll be better to intruduce a separate util file for it?
I'll leave it here. it kinda belongs cause web3 has the method to create a web3 contract instance from abi and address and this one does the same. I kinda see your point, but not to the extend to actually changing it.

Reply:
Since we are planning on moving `web3Wrapper` out into it's own subpackage, and re-using it amongst `contracts` and `0x.js` and potentially `website` it might make sense to do eventually. Feel free to punt for now.